weaver/common/protos-go/common/access_control.pb.go

```diff
@@ -1,3 +1,7 @@
+// Copyright IBM Corp. All Rights Reserved.
```

Comment:
I think the copyright statements should be removed like it's done in other weaver generated protos

Reply:
@outSH Let's skip this as we discussed in the last Cacti maintainers' call.